README.md

```diff
@@ -1,85 +1,78 @@
-# About
-This library mocks VK Connect methods from [vkui-connect](https://www.npmjs.com/package/@vkontakte/vkui-connect) for developing in desktop browser
+# VK Connect Mock
 
-# How to install
-Add library to your project dependencies
+<img width="100" height="100" src="https://avatars3.githubusercontent.com/u/1478241?s=200&v=4" align="right">
 
-`npm install @vkontakte/vkui-connect-mock`
+This library mocks [VK Connect](https://www.npmjs.com/package/@vkontakte/vk-connect) methods.
 
-or
+## Usage
 
-`yarn add @vkontakte/vkui-connect-mock`
+Install the library via yarn
 
-# How to use
+```
+$ yarn add @vkontakte/vk-mini-apps-api
+```
 
-Import necessary methods from this library instead of using [vkui-connect](https://www.npmjs.com/package/@vkontakte/vkui-connect):
+or npm
 
-``` javascript
-import VKConnect from '@vkontakte/vkui-connect-mock';
 ```
+$ npm install @vkontakte/vk-mini-apps-api
+```
+
+Use in your code instead of using [vk-connect](https://www.npmjs.com/package/@vkontakte/vk-connect) by following way:
 
-And then init VK Connect:
-``` javascript
-VKConnect.subscribe((e) => {
+```javascript
+import vkConnect from '@vkontakte/vk-connect-mock';
+
+vkConnect.sendPromise('VKWebAppGetUserInfo', {}).then(data => {
+  // Do something
+});
+```
+
+Or event-based way:
+
+```javascript
+import vkConnect from '@vkontakte/vk-connect-mock';
+
+vkConnect.subscribe(e => {
   if (e.detail.type === 'VKWebAppGetUserInfoResult') {
-    // do something
+    // Do something
   }
 });
-VKConnect.send('VKWebAppGetUserInfo', {});
+
+vkConnect.send('VKWebAppGetUserInfo', {});
 ```
-More documentation regarding VK Connect [is here](https://vk.com/dev/vk_apps_docs?f=4.%20%D0%9F%D0%BE%D0%B4%D0%BA%D0%BB%D1%8E%D1%87%D0%B5%D0%BD%D0%B8%D0%B5%20VK%20Connect).
 
-# How to edit mocked data
-There are getter and setter for mocked data of each method.
-For example, you want to change response of `VKWebAppGetUserInfo` method:
-``` javascript
-import { response as res } from  '@vkontakte/vkui-connect-mock';
-res.VKWebAppGetUserInfo.data = {
-  type: 'VKWebAppGetUserInfoResult',
-  data: {
-    photo_100: 'https://pp.userapi.com/c841034/v841034569/3b8c1/pt3sOw_qhfg.jpg',
-    photo_200: 'https://pp.userapi.com/c841034/v841034569/3b8c1/pt3sOw_qhfg.jpg',
-    first_name: 'Friedrich',
-    last_name: 'Engels',
-    sex: 0,
-    city: {
-      id: 1,
-      title: 'London',
-    },
-    country: {
-      id: 1,
-      title: 'UK'
-    },
-    timezone: 0
-  },
-}
+Please note that some methods may only receive (for example, `VKWebAppUpdateConfig`,
+`VKWebAppViewHide`, `VKWebAppViewRestore`, etc.). To obtain data from them you need to use the following function:
```

Comment:
To obtain data from them you need to use the event-based way and callReceiveOnlyMethod()